__tests__/formats/typeScriptEs6Declarations.test.js

```diff
@@ -40,7 +38,7 @@ describe('formats', () => {
         dictionary,
         file,
         platform: {},
-      }), {}, file);
```

Comment:
I removed the 2nd and 3rd argument here because createFormatArgs only has the one options parameter.